components/CampaignListItem/index.tsx

```diff
@@ -0,0 +1,139 @@
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined'
+import { Box } from '@mui/material'
+import { CampaignStatus } from 'constants/sidebarData'
+
+const styledmainbox = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '50%',
+  height: '100%',
+  padding: '10px',
+  backgroundColor: '#EAF9FF',
+}
+
+const styledmainbody = {
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledcontentbox = {
+  display: 'flex',
+  height: '300px',
+  backgroundColor: '#D9D9D9',
+  padding: '15px',
+}
+
+const styledbottombox = {
+  display: 'flex',
+  flex: 'auto',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  height: '67px',
+  backgroundColor: '#FFFFFF',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledbottomitems = {
+  display: 'flex',
+}
+
+const styledinput = {
+  fontSize: '15',
+  fontWeight: '700',
+}
+
+const styledoverview = {
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+} as React.CSSProperties
+
+const campaignStateColor = {
+  [CampaignStatus.PUBLISHED]: '#16A64A',
+  [CampaignStatus.DRAFT]: '#C7043D',
+  [CampaignStatus.ARCHIVED]: '#641869',
+}
+
+interface Props {
+  campaignName: string
+  campaignDetails: string
+  donors: number
+  amount: number
+  country: string
+  campaignStatus: CampaignStatus
+  viewAll: boolean
+}
+
+const CampaignListItem = ({
+  campaignName,
+  campaignDetails,
+  donors,
+  amount,
+  country,
+  campaignStatus,
+  viewAll,
+}: Props) => {
+  return (
+    <Box sx={styledmainbox}>
+      <Box sx={styledmainbody}>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+            flexDirection: 'row',
+          }}
+        >
+          <h3 style={{ color: '#009DD7' }}>{campaignName}</h3>
+          <ArchiveOutlinedIcon
+            sx={{
+              color: '#009DD7',
+              fontSize: '2.5rem',
+            }}
+          />
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            flexDirection: 'row',
+            alignItems: 'center',
+          }}
+        >
+          <p style={styledoverview}>{campaignDetails}</p>
+          <h4
+            style={{
+              color: campaignStateColor[campaignStatus],
+              marginLeft: 50,
+            }}
+          >
+            {campaignStatus}
+          </h4>
+        </div>
+      </Box>
+      <div>
+        {viewAll ? <Box sx={styledcontentbox}>{campaignDetails}</Box> : <p></p>}
+      </div>
```

Comment:
no need to wrap in `div` and can just be `{viewAll && <Box sx={styledcontentbox}>{campaignDetails}</Box>}`